tic_tac_toe/game.py

```diff
@@ -63,22 +74,31 @@ def _check_winning_combinations(board, player):
     Returns the player (winner) of any of the winning combinations is completed
     by given player, or None otherwise.
     """
-    pass
+    for i in range(3):
+        if _is_winning_combination(board, ((i,0), (i,1), (i,2)), player) or _is_winning_combination(board, ((0,i), (1,i), (2,i)), player):
+            return player
+    
+    if _is_winning_combination(board, ((0,0), (1,1), (2,2)), player) or _is_winning_combination(board, ((0,2), (1,1), (2,0)), player):
+        return player
+        
+    return None
```

Comment:
Good use of for loop rather than hardcoding every winning combination and checking against one by one.